src/components/MAPI/utils.ts

```diff
@@ -181,6 +183,7 @@ export async function fetchMasterListForCluster(
         labelSelector: {
           matchingLabels: {
             [capiv1alpha3.labelCluster]: cluster.metadata.name,
+            [capzv1alpha3.labelControlPlane]: 'true',
```

Comment:
In the current REST API logic, this is also necessary in order for an `AzureMachine` to be a Control Plane node.